src/app/app.module.ts

```diff
@@ -16,6 +19,14 @@ import { InventoryData } from '../providers/inventory-data';
 import { StockpileData } from '../providers/stockpile-data';
 import { UserData } from '../providers/user-data';
 
+let storage = new Storage();
+
+export function getAuthHttp(http) {
+  return new AuthHttp(new AuthConfig({
+    tokenGetter: (() => storage.get('id_token')),
```

Comment:
I'm not sure why, but it seems to be the standard to name the auth token `id_token`. It is that name default, it simply has to be overwritten to use ionic storage.